src/translations/en.json

```diff
@@ -6816,7 +6816,7 @@
           "forums": "Home Assistant forums",
           "open_home_newsletter": "Building the Open Home newsletter",
           "discord": "Discord chat",
-          "twitter": "Twitter",
+          "twitter": "X (formerly Twitter)",
```

Comment:
We can also change the key to `x`. It will invalidate the translation for all languages.